src/views/open-api-docs-view/index.tsx

```diff
@@ -49,9 +51,16 @@ function OpenApiDocsView({
 					className={s.overview}
 					title={title}
 					badgeText={releaseStage}
-					description={description}
+					contentSlot={
+						descriptionMdx ? (
+							<DescriptionMdx mdxRemoteProps={descriptionMdx} />
+						) : null
+					}
 				/>
-				<OpenApiOperations operationGroups={operationGroups} />
+				<div className={s.operationsSection}>
+					<h2 className={s.operationsHeading}>Operations</h2>
```

Comment:
I added this `Operations` heading to ensure that the `h3` for each operation doesn't end up semantically nested under any of the provided `description` markdown content.

Note: in designs this heading is `Services`, I think `Operations` is more technically correct since unlike previous API docs (such as Boundary's), HCP API docs can't be organized into services since `HCP Vault Secrets` is itself the service.